### DISCUSSION TITLE
Live Component issue renders multiple times

### DISCUSSION THREAD

**Discussion by G-Mein** [symfony/symfony#58606] [Q&A] [answered]
I got a very small page with news and events I am right now building. The news have fixtures with 40 items and both are on different pages. switched through the navbar.

I built a pagination and newslist live component for loading the news on the frontpage.

Now when I click on the news and then switch to the "events" link it loads the page through ajax and works. When I then return to news and use the pagination, there are multiple event listeners registered on the pagination links. And also on the "read more" links on the news cards itself. 

As I said, very simple, not productive just for testing UX Turbo and live components for self education

anyone had this Isse themself and knows how to fix this and where this comes from?

**Comment by G-Mein** [comment]
I now found out, when I hove over some specific links, the profiler shows me, that the AJAX requests are already done. But only on the pages I am not on. The more I hover, the more ajax requests are done afterwards.

**Comment by smnandre** [comment]
It's a Turbo thing.. https://turbo.hotwired.dev/handbook/drive#prefetching-links-on-hover

**Comment by G-Mein** [comment]
thx, the hover prefetch is off but the problem sadly still occurs with the multiple ajax requests sent out.

**Reply by G-Mein** [reply]
it now only counts up, when i visit the pages of course

**Reply by G-Mein** [reply, marked as answer]
digging deeper in the ux turbo doc I found the point, where to fully reload the page. With that meta enabled on every page, I don't have the problem anymore. Maybe it seems anyway to be a turbo problem. I will make an issue on their page. But with full reload enabled, I can now use the live action stuff on some pages like a charm. thx for the help!
